<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>487</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border-radius: 10px;
	background-color: #1B1D23;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;Sitka Small&quot;;
    border: none;
    border-radius: 10px;
	background-color:  #3d3a59;
    color: rgb(250, 250, 250);
}
QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton:pressed{
	background-color: #604d94;
}
</string>
     </property>
     <property name="text">
      <string>Информация</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;Sitka Small&quot;;
    border: none;
    border-radius: 10px;
	background-color:  #3d3a59;
    color: rgb(250, 250, 250);
}
QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton:pressed{
	background-color: #604d94;
}

</string>
     </property>
     <property name="text">
      <string>Примкнуть</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;Sitka Small&quot;;
    border: none;
    border-radius: 10px;
	background-color:  #3d3a59;
    color: rgb(250, 250, 250);
}
QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton:pressed{
	background-color: #604d94;
}
</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;Sitka Small&quot;;
    border: none;
    border-radius: 10px;
	background-color:  #3d3a59;
    color: rgb(250, 250, 250);
}
QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton:pressed{
	background-color: #604d94;
}
</string>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font: 12pt &quot;Sitka Small&quot;;
    border: none;
    border-radius: 10px;
	background-color:  #3d3a59;
    color: rgb(250, 250, 250);
}
QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton:pressed{
	background-color: #604d94;
}
</string>
     </property>
     <property name="text">
      <string>Переводчик</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-radius: 15px;
	background-color: #fff;
    font: 12pt &quot;Sitka Small&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>631</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 15px;
	background-color: #3d3a59;
}
QListWidget {
	font: 12pt &quot;Sitka Small&quot;;
    border: none;
    border-radius: 10px;
	background-color:  #3d3a59;
    color: rgb(250, 250, 250);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>370</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;
     border-radius: 10px;
    background-color:  #3d3a59;
}
QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton:pressed{
	background-color: #604d94;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>781</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 10px;
	background-color:  #3d3a59;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>744</x>
        <y>0</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton{
    border:none
}
QPushButton:pressed{
	background-color: #604d94;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>704</x>
        <y>0</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
}
QPushButton:hover{
	background-color: #4a3c73;
}
QPushButton:pressed{
	background-color: #604d94;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>